### PR TITLE
[DEV-10892] - Add filters flags list sessions

### PR DESCRIPTION
**Pull request by tomerstein**

### CODE REVIEW

File: pkg/commands/access/actions/list.go [hunk 1]
```diff
@@ -39,5 +54,49 @@ func AccessList() *cobra.Command {
 		},
 	}
 
+	flags := cmd.Flags()
+	flags.StringVarP(&integrationFilter, integrationFilterFlagName, "i", "", "The integration id or type/name, for example: \"aws-account/My AWS integration\"")
```
Comment (DmitriyO) [inline pkg/commands/access/actions/list.go:58]: the type the user should use it the type id? is it exposed to him anywhere? ususally we show the display name to the user

File: pkg/commands/access/actions/list.go [hunk 2]
```diff
@@ -39,5 +54,49 @@ func AccessList() *cobra.Command {
...
+	flags.StringVarP(&bundleFilter, bundleFilterFlagName, "b", "", "filter by bundle name or id")
```
Comment (DmitriyO) [inline pkg/commands/access/actions/list.go:59]: do we really want bundle id? I mean will the users ever use that?

File: pkg/services/sessions.go [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+	"context"
+
+	"github.com/apono-io/apono-cli/pkg/aponoapi"
+	"github.com/apono-io/apono-cli/pkg/clientapi"
+	"github.com/apono-io/apono-cli/pkg/utils"
+)
+
+func ListAccessSessions(ctx context.Context, client *aponoapi.AponoClient, integrationIds []string, bundleIds []string) ([]clientapi.AccessSessionClientModel, error) {
+	return utils.GetAllPages(ctx, client, func(ctx context.Context, client *aponoapi.AponoClient, skip int32) ([]clientapi.AccessSessionClientModel, *clientapi.PaginationClientInfoModel, error) {
+		listBundleRequest := client.ClientAPI.AccessSessionsAPI.ListAccessSessions(ctx).Skip(skip)
```
Comment (DmitriyO) [inline pkg/services/sessions.go:13]: -> listSessionsReqeust?

File: pkg/commands/access/actions/list.go [hunk 4]
```diff
@@ -39,5 +54,49 @@ func AccessList() *cobra.Command {
...
+	}
+
+	if utils.IsValidUUID(integrationIDOrName) {
+		return []string{integrationIDOrName}
+	}
+
+	integration, err := services.GetIntegrationByIDOrByTypeAndName(ctx, client, integrationIDOrName)
+	if err != nil {
+		return []string{integrationIDOrName}
+	}
```
Comment (DmitriyO) [inline pkg/commands/access/actions/list.go:99]: same here

**Review by DmitriyO — CHANGES_REQUESTED** [review]
Cool, some small comments

File: pkg/commands/access/actions/list.go (hunk 1 above)
Comment (tomerstein) [reply pkg/commands/access/actions/list.go:58]: its the type ID and it exposed in the `apono inventory integrations` command

File: pkg/commands/access/actions/list.go (hunk 2 above)
Comment (tomerstein) [reply pkg/commands/access/actions/list.go:59]: thats what we talked about and its in the PRD

File: pkg/commands/access/actions/list.go [hunk 5]
```diff
@@ -39,5 +54,49 @@ func AccessList() *cobra.Command {
...
+	bundles, err := services.ListBundles(ctx, client, bundleIDOrName)
+	if err != nil {
+		return []string{bundleIDOrName}
+	}
+
+	for _, bundle := range bundles {
+		if bundle.Id == bundleIDOrName || bundle.Name == bundleIDOrName {
+			return []string{bundle.Id}
+		}
+	}
```
[the remark this answers is not in the document]
Comment (tomerstein) [reply pkg/commands/access/actions/list.go:82]: you are right, there is no need to check the id later, will remove the later check

File: pkg/commands/access/actions/list.go (hunk 1 above)
Comment (DmitriyO) [reply pkg/commands/access/actions/list.go:58]: ok

File: pkg/commands/access/actions/list.go (hunk 2 above)
Comment (DmitriyO) [reply pkg/commands/access/actions/list.go:59]: 👍

**Review by DmitriyO — APPROVED** [review]

**Pull request merged**